config/services_test.yaml

```diff
@@ -0,0 +1,3 @@
+serivices:
+  App\Issues\GitHub\CachedLabelsApi:
+    class: App\Tests\Service\Issues\Github\FakedCachedLabelsApi
```

Comment:
This has a typo - the actual class has no `s` on `Labels`.

Reply:
Thank you